pkg/cmd/admin/network/isolate_project.go

```diff
@@ -0,0 +1,77 @@
+package network
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	kerrors "k8s.io/kubernetes/pkg/util/errors"
+
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+)
+
+const (
+	IsolateProjectNetworkCommandName = "isolate-projects"
+
+	isolateProjectNetworkLong = `
+Isolate project network
+
+Allows project to isolate their network from other projects.`
```

Comment:
I'll fix this one. I need to check how we can get network plugin name in pod-network command, that way we can provide useful error message when this cmd is used for flat sdn.